make_prg/update/denovo_variants.py

```diff
@@ -43,6 +50,11 @@ def _param_checking(start_index_in_linear_path: int, ref: str, alt: str):
         if negative_index_for_variant_pos:
             raise DenovoError(f"Found a negative index for variant pos ({start_index_in_linear_path})")
 
+        deletion_size = len(ref) - len(alt)
```

Comment:
So insertions don't cause the same issues?

Reply:
Not that I know of. I had a branch that would ignore any indels, but the issues were really caused by deletions only (which makes sense from an algorithmic point of view)